react/__tests__/components/BuyButton.test.tsx

```diff
@@ -19,7 +20,7 @@ const mocks = [
 ]
 
 describe('<BuyButton />', () => {
-  const renderComponent = (customProps, text = 'Test') => {
+  const renderComponent = (customProps: any, text = 'Test') => {
```

Comment:
Add optional for the second argument too
```suggestion
  const renderComponent = (customProps?: any, text = 'Test') => {
```